.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: test
+
+on: [push]
```

Comment:
on:
  push:
    branches:
      - master
      - next
  pull_request:
    branches:
      - master
      - next

Reply:
why do you want to specify the two branches?

Reply:
Because we should run on pull_request too, we always use next for next major release, so when we start to work the next time on a major release, we have to add this, we can avoid it and add it initially